### ISSUE TITLE
Delete ENV variable fails in Windows

### ISSUE THREAD

**Issue by MRod15** [cucumber/aruba#349]
## Summary

Deleting ENV variables is failing in Windows with the error:
`wrong number of arguments (1 for 2) (ArgumentError)`

*No backtrace is given!*

## Expected Behavior

Variable should be removed from ENV.

## Current Behavior

see _Steps to Reproduce (for bugs)_

## Possible Solution

Followed the error to:

> {ArgumentError} wrong number of arguments (1 for 2)
> "C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/platforms/windows_environment_variables.rb:84:in `delete'"
> "C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/api/deprecated.rb:406:in `block in restore_env'"
> "C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/api/deprecated.rb:401:in `each'"
> "C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/api/deprecated.rb:401:in `restore_env'"
> "C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/cucumber/hooks.rb:19:in `After'"

and found that the _*delete*_ method, which is inherited from _*UnixEnvironmentVariables*_, has an extra argument that its _*super*_.

## Steps to Reproduce (for bugs)

```
irb(main):001:0> require 'aruba'
=> true
irb(main):003:0> env = Aruba::Platforms::WindowsEnvironmentVariables.new
=> ...
irb(main):004:0> env['foo']='bar'
=> "bar"
irb(main):005:0> env['foo']
=> "bar"
irb(main):006:0> env.delete 'foo'
ArgumentError: wrong number of arguments (1 for 2)
        from C:/Ruby22/lib/ruby/gems/2.2.0/gems/aruba-0.14.0/lib/aruba/platforms/windows_environment_variables.rb:84:in `delete'
        from (irb):6
        from C:/Ruby22/bin/irb:11:in `<main>'
```

## Context

Scenario fails "silently" after steps in an Aruba hook (After) afecting test results.

## Environment
* Aruba 0.14.0 @ Ruby 2.2.0
* Windows 8.1

**Comment by dg-ratiodata** [comment]
Thanks for reporting this. This should be fixed when #350 is merged.

**Comment by dg-ratiodata** [comment]
Should be fixed in master.
